docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -83,94 +82,83 @@ From the Azure portal, create a client secret. Rancher will use this key to auth
 
      ![Open Rancher Registration](/img/open-rancher-app-reg.png)
 
-1. From the navigation pane on left, click **Certificates and Secrets**.
+1. From the navigation pane on left, click **Certificates & secrets**.
 
 1. Click **New client secret**.
     ![Create new client secret](/img/new-client-secret.png)
 1. Enter a **Description** (something like `Rancher`).
 1. Select duration for the key from the options under **Expires**. This drop-down sets the expiration date for the key. Shorter durations are more secure, but require you to create a new key after expiration.
+Note that users won't be able to log in to Rancher when it detects that the application secret has expired, so rotate the secret in Azure and update in Rancher before it expires.
```

Comment:
I like these special notes and warnings, but I am trying to avoid excessive formatting, which can make the docs too dense.
There is so much going on already with bold, italic, underscores, links, etc.